xep-0369.xml

```diff
@@ -80,11 +85,11 @@
 
 <section1 topic='Requirements' anchor='reqs'>
   <ul>
-    <li>A user's participation in a channel persists and is not modified by the user's client going online and offline.</li>
+    <li>A user's participation in a channel may persist and is not modified by the user's client going online and offline.</li>
```

Comment:
As far as I can tell this is not mandatory; added a "may" (non RFC 2119 style since this is just a requirements section) to clarify. Please advise if this is incorrect.